CHANGELOG.rst

```diff
@@ -4,6 +4,14 @@ Change Log
 v3.x
 ----
 
+v3.0.0-beta.5 (Not released yet)
```

Comment:
Can you revert the changes to the change log? I add these changes only on the `master` branch to avoid merge conflicts. You can draft the change log entry in a PR comment.

Reply:
Done.